lib/chromedriver.js

```diff
@@ -251,14 +251,18 @@ class Chromedriver extends events.EventEmitter {
 
     if (this.adb) {
       log.debug(`Cleaning any old adb forwarded port socket connections`);
-      for (let conn of await this.adb.getForwardList()) {
-        // chromedriver will ask ADB to forward a port like "deviceId tcp:port localabstract:webview_devtools_remote_port"
-        if (conn.indexOf('webview_devtools') !== -1) {
-          let params = conn.split(/\s+/);
-          if (params.length > 1) {
-            await this.adb.removePortForward(params[1].replace(/[\D]*/, ''));
+      try {
```

Comment:
shouldn't we only cover this.adb.getForwardList() call with try/catch?

Reply:
The we need the variable outside and whatnot. I don't see the point in just having the call, and not the processing. Also, what happens if `adb.removePortForward` fails?

Reply:
>  Also, what happens if adb.removePortForward fails?

then the warning message could be updated to reflect this case as well